perun-core/src/main/resources/schema.sql

```diff
@@ -164,6 +164,18 @@ create table facility_contacts (
 	group_id integer
 );
 
+create table facility_services (
```

Comment:
There is also ```test-data.sql``` file, where you need to update DB version in order for tests to run correctly.